sunpy/visualization/animator/base.py

```diff
@@ -435,51 +435,29 @@ def __init__(self, data, image_axes=[-2, -1], axis_ranges=None, **kwargs):
             slider_axes.remove(x)
 
         if len(slider_axes) != self.num_sliders:
-            raise ValueError("Specific the same number of axes as sliders!")
+            raise ValueError("Number of sliders doesn't match number of slider axes.")
```

Comment:
`Number of sliders doesn't match the number of slider axes.`

Reply:
Done